linux_os/guide/services/ntp/var_multiple_time_servers.var

```diff
@@ -14,3 +14,4 @@ options:
     rhel: "0.rhel.pool.ntp.org,1.rhel.pool.ntp.org,2.rhel.pool.ntp.org,3.rhel.pool.ntp.org"
     ol: "0.pool.ntp.org,1.pool.ntp.org,2.pool.ntp.org,3.pool.ntp.org"
     suse: "0.suse.pool.ntp.org,1.suse.pool.ntp.org,2.suse.pool.ntp.org,3.suse.pool.ntp.org"
+    aliyun: "0.ntp.cloud.aliyuncs.com,1.ntp.aliyun.com,2.ntp1.aliyun.com,3.ntp1.cloud.aliyuncs.com"
```

Comment:
Since the product namesare [alinux2](https://github.com/ComplianceAsCode/content/blob/master/products/alinux3/product.yml) and [alinux3](https://github.com/ComplianceAsCode/content/blob/master/products/alinux3/product.yml) please use `alinux` here.
```suggestion
    alinux: "0.ntp.cloud.aliyuncs.com,1.ntp.aliyun.com,2.ntp1.aliyun.com,3.ntp1.cloud.aliyuncs.com"
```